Framework/Geometry/test/ComponentInfoTest.h

```diff
@@ -108,10 +108,11 @@ std::unique_ptr<Beamline::ComponentInfo> makeSingleBeamlineComponentInfo(
   using Mantid::Beamline::ComponentType;
   auto isStructuredBank =
```

Comment:
Not your changes but, `isStructured` bank is a bad name for this variable now. `componentType` would be better. Change if you have time.